back/src/domain/model/Activity.ts

```diff
@@ -9,21 +9,30 @@ export class Activity {
     private _project: Project;
     private _matin: boolean;
     private _date: Date;
-    private _cra: CRA;
+    private _craId: number;
+    toJSON(): object {
```

Comment:
Why is this method in the Domain ? Why do we even need it as JSON ?
You can create a mapper or something like that in the "application" layer not the domain